.devcontainer/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM mcr.microsoft.com/vscode/devcontainers/base:0-ubuntu20.04
+
+ARG BUILD_ELASTIC_STACK_VERSION="7.10.2"
+ARG BUILD_TERRAFORM_VERSION="1.0.9"
+ARG BUILD_MITMPROXY_VERSION="7.0.4"
+
+# Update & Install Open JDK
+RUN apt-get update && apt-get upgrade -y && apt-get install -y default-jre
+
+# Install Terraform
+ENV TERRAFORM_VERSION=${BUILD_TERRAFORM_VERSION}
+RUN curl -Os https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip && \
+    unzip -o terraform_${TERRAFORM_VERSION}_linux_amd64.zip -d /usr/local/bin
+RUN rm terraform_${TERRAFORM_VERSION}_linux_amd64.zip
+
+# Install Azure CLI
+RUN curl -sL https://aka.ms/InstallAzureCLIDeb | bash
+
+# Install Helm
+RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+RUN chmod 700 get_helm.sh
+RUN ./get_helm.sh
+
+# Install mitmproxy
+ENV MITMPROXY_VERSION=${BUILD_MITMPROXY_VERSION}
+RUN wget https://snapshots.mitmproxy.org/${MITMPROXY_VERSION}/mitmproxy-${MITMPROXY_VERSION}-linux.tar.gz
+RUN tar -xzf mitmproxy-${MITMPROXY_VERSION}-linux.tar.gz -C /usr/local/bin
+RUN rm mitmproxy-${MITMPROXY_VERSION}-linux.tar.gz
```

Comment:
why do we need this?

two options as I see it:
a) leave it and provide info how to use it in the dev container docs
b) remove it

Reply:
OK, will work on option a).

Reply:
@eladiw @tomconte Just added documentation page for mitmproxy. In addition, I removed dotnet-sdk-3.1 and added dotnet-sdk-6.0 instead (so this base image contains dotnet-sdk 5.0 and 6.0).